scripts/nic-pci2dut

```diff
@@ -12,6 +12,17 @@ NIC_MODELS["1924:0a03"]="medford"
 # Solarflare Communications XtremeScale SFC9250 10/25/40/50/100G Ethernet Controller
 NIC_MODELS["1924:0b03"]="medford"
 
+# Mellanox Technologies MT27800 Family [ConnectX-5]
```

Comment:
There are two patches: one on renaming and one on adding new adapters, please, make them separate.